src/modules/stats/overwatch-stats.ts

```diff
@@ -0,0 +1,189 @@
+import { stripIndents } from 'common-tags';
+import { Message, MessageEmbed } from 'discord.js';
+import { Command, CommandoMessage, CommandoClient } from 'discord.js-commando';
+import { startTyping, stopTyping, deleteCommandMessages } from '../../lib/helpers';
+import { getProfile, Profile, PLATFORM, REGION } from 'overwatch-api';
+
+class OverwatchStatsArguments { 
+	battletag: string; 
+	platform: PLATFORM;
+	region: REGION;
+}
+/**
+ * Post information about an alcoholic brew.
+ *
+ * @export
+ * @class OverwatchStatsCommand
+ * @extends {Command}
+ */
+export default class OverwatchStatsCommand extends Command {
+	/**
+	 * Creates an instance of BrewCommand.
+	 *
+	 * @param {CommandoClient} client
+	 * @memberof BrewCommand
+	 */
+	constructor(client: CommandoClient) {
+		super(client, {
+			args: [
+				{
+					key: 'battletag',
+					parse: (battletag: string): string => battletag.replace('#', '-'),
+					prompt: 'Enter the battletag (case-sensitive)?',
+					type: 'string'
+				},
+				{
+					default: 'pc',
+					key: 'platform',
+					parse: (platform: string): string => platform.toLowerCase(),
+					prompt: 'Choose a platform: pc | xbl | psn',
+					type: 'string',
+					validate: (platform: string): boolean | string => {
+						const options: Array<string> = ['pc', 'xbl', 'psn'];
+						if (options.indexOf(platform) === -1) {
+							return `Platform must be one of the following: ${options.join(', ')}.`
+						}
+						
+						return true;
+					}
+				},
+				{
+					default: 'us',
+					key: 'region',
+					parse: (region: string): string => region.toLowerCase(),
+					prompt: 'Choose a region: us | eu | kr | cn | global',
+					type: 'string',
+					validate: (region: string) => {
+						const options: Array<string> = ['us', 'eu', 'kr', 'cn', 'global'];
+						if (options.indexOf(region) === -1) {
+							return `Region must be one of the following: ${options.join(', ')}.`
+						}
+						
+						return true;
+					}
+				}
+			],
+			description: 'Returns overwatch stats about a player.',
+			details: stripIndents`
+				syntax: \`!overwatch-stats <battletag> <platform: pc|xb1|psn> <region: eu|us>\`
+			`,
+			examples: [
+				'!overwatch-stats Mythos-11321',
+				'!overwatch-stats Mythos-11321 pc',
+				'!overwatch-stats Mythos-11321 pc us'
+			],
+			group: 'stats',
+			guildOnly: true, 
+			memberName: 'overwatch-stats',
+			name: 'overwatch-stats',
+			throttling: {
+				duration: 3,
+				usages: 2
+			}
+		});
+	}
+
+	/**
+	 * Run the "overwatch-stats" command.
+	 *
+	 * @param {CommandoMessage} msg
+	 * @param {OverwatchStatsArguments} args
+	 * @returns {(Promise<Message | Message[]>)}
+	 * @memberof BrewCommand
+	 */
+	public async run(msg: CommandoMessage, args: OverwatchStatsArguments): Promise<Message | Message[]> {
+		startTyping(msg);
+
+		// Replace any hashtag with a dash for proper api formatting.
+		args.battletag = args.battletag.replace('#', '-');
```

Comment:
Remove this in favor of the same call on line [#111](https://github.com/Spudnik-Group/Spudnik/pull/213/commits/7593783dca68a3b209814b701ab8867a08424678#diff-3311edac79d99f31856505ba74fa5781R111).

Reply:
Just to document here, I make sure the api gets it with a dash as that is how the api expects it, and I show it with a hashtag since that is how battlenet displays it.